lib/purserl.ex

```diff
@@ -1769,25 +1770,32 @@ defmodule DevHelpers.Purserl do
     end
   end
 
-  defp ioputs(device \\ :stdio, item) do
-    IO.puts(device, item)
```

Comment:
I assume this was an example of "I don't understand why this was here, so I'm going to remove it". It does look weird, right? It's just a wrapper with a default argument that matches the default argument of `IO.puts`. Who in their right mind would make a wrapper function like this? It's odd, right? But my coworkers aren't stupid, so there might be a good reason why it was here. Let's at least ask them to double-check this weird thing we found, right?

Reply:
That is fair, but this just leaves me wondering what was the use of this function?

Reply:
Yep. I should've commented the why, but I didn't. And now you asked, which is what I want you to do when you see code you don't understand the purpose of. The purpose of this indirection was to, when developing, allow 1. disabling all prints at once, and 2. redirect prints to File.write instead of IO.puts.